### ISSUE TITLE
replace querystring

### ISSUE THREAD

**Issue by aindrajaya** [sebs/etherscan-api#112]
### Description:
Deprecated libs (`querystring`)

### Code to reproduce:
```js
//this one
var query = querystring.stringify({
  mod, action, apiKey, bool
})

//change into this
const dat = new URLSearchParams({
  mod, action, apiKey, bool
}).toString()
```

## Reason
`querystring` was deprecated at least since last LTS version 14.

cc @sebs

**Comment by sebs** [comment]
erm .. yeah ;) sounds like a plan

**Comment by aindrajaya** [comment]
Sure, can I make a pull request?

**Comment by sebs** [comment]
sure .. do so

**Comment by sebs** [comment]
fixed in 10.3.0
